native/mod_manager/mod_manager.c

```diff
@@ -723,6 +695,16 @@ static int manager_init(apr_pool_t *p, apr_pool_t *plog,
      */
     mc_initialize_cleanup(p);
 
+    /* Create global mutex */
+    if (ap_global_mutex_create(&node_mutex, NULL, node_mutex_type, NULL, s, p, 0) != APR_SUCCESS) {
+        ap_log_error(APLOG_MARK, APLOG_ERR|APLOG_NOERRNO, 0, s,"manager_init: ap_global_mutex_create failed");
+        return !OK;
+    }
+    if (ap_global_mutex_create(&context_mutex, NULL, context_mutex_type, NULL, s, p, 0) != APR_SUCCESS) {
+        ap_log_error(APLOG_MARK, APLOG_ERR|APLOG_NOERRNO, 0, s,"manager_init: ap_global_mutex_create failed");
```

Comment:
I would consider adding which mutex failed (node/context) as the text is the same for both. The same applies for the `apr_global_mutex_child_init`.

Reply:
apr_global_mutex_child_init already log the node_mutex_type or context_mutex_type